sycl/test-e2e/KernelFusion/queue-shortcut-functions.cpp

```diff
@@ -2,6 +2,9 @@
 // RUN: env SYCL_RT_WARNING_LEVEL=1 %{run} %t.out 2>&1 \
 // RUN:   | FileCheck %s --implicit-check-not=ERROR
 
+// https://github.com/intel/llvm/issues/15275
+// XFAIL: gpu-intel-gen12 || gpu-intel-dg2
```

Comment:
The problem seems to be that we expect debug output to be in consecutive lines, but `ZE_LOADER_DEBUG_TRACE` output is interleaved with what we expect. 

I'm fine with XFAILing.